package.json

```
@@ -30,6 +30,7 @@
     ".eslintrc.yaml"
   ],
   "scripts": {
+    "commit:lint": "commitlint --from=main --to=HEAD",
```

I find `commit` ambiguous in a script name, as it looks like a verb.
```suggestion
    "commit-messages:lint": "commitlint --from=main --to=HEAD",
```